base/exports.jl

```diff
@@ -1409,6 +1411,7 @@ export
 
     @assert,
     @enum,
+    @flagenum,
```

Comment:
new exports need to be listed in a docs index to go into the new manual I believe